ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BeaconStateUtil.java

```diff
@@ -126,7 +189,7 @@ public static long get_current_epoch(BeaconState state) {
 
   public static long get_previous_epoch(BeaconState state) {
     if (UnsignedLong.valueOf(get_current_epoch(state))
-            .compareTo(slot_to_epoch(Constants.GENESIS_SLOT))
+            .compareTo(slot_to_epoch(UnsignedLong.valueOf(Constants.GENESIS_SLOT)))
```

Comment:
Can we replace with:
```
if ( get_current_epoch(state) > Constants.GENESIS_EPOCH )
```

Reply:
That's what it originally read :-)

However, Joe had already done some work to prepare for epochs being UnsignedLong and I didn't want to completely undo it. Once epochs are transitioned, it will need to look like this anyway.